clang/test/SemaSYCL/intel-fpga-loops.cpp

```diff
@@ -66,6 +68,11 @@ void foo_deprecated() {
   // expected-note@+1 {{did you mean to use 'intel::speculated_iterations' instead?}}
   [[intelfpga::speculated_iterations(6)]] for (int i = 0; i != 10; ++i)
       a[i] = 0;
+
+  // expected-warning@+2 {{attribute 'intelfpga::loop_control_avg' is deprecated}}
+  // expected-note@+1 {{did you mean to use 'intel::loop_control_avg' instead?}}
+  [[intelfpga::loop_control_avg(6)]] for (int i = 0; i != 10; ++i)
+      a[i] = 0;
```

Comment:
This is not needed since this is new attribute and we do not use any intelfpga spelling